web/src/settings_config.ts

```diff
@@ -919,34 +919,42 @@ export const desktop_icon_count_display_values = {
 export const system_user_groups_list = [
     {
         name: "role:internet",
+        dropdown_option_name: $t({defaultMessage: "Everyone on the internet"}),
         display_name: $t({defaultMessage: "Everyone on the internet"}),
     },
     {
         name: "role:everyone",
-        display_name: $t({defaultMessage: "Admins, moderators, members and guests"}),
+        dropdown_option_name: $t({defaultMessage: "Admins, moderators, members and guests"}),
```

Comment:
Maybe we can name this field something like "description".

Reply:
Yeah, we can see; we may find we end up deleting it after migrating all the groups inputs to the modern format, so probably not worth effort on its name at this time.